styles.css

```diff
@@ -60,18 +60,27 @@ section {
 }
 
 .footer-links {
-    overflow: auto;
-    position: relative;
+    overflow: hidden;
+    position: absolute;
+    display: inline-block;
     width: 100%;
     height: 30vh;
 }
 
+.footer-nav-bar {
+    float: left;
+    margin: 0 auto;
+    width: 25%;
+    border: 5px white;
+    box-sizing: border-box;
```

Comment:
would it work just having box-sizing: border-box; on the body tag?

Reply:
Yes ! it would ! I've deleted unnecessary box-sizing tags